### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by paulbjensen**
- Typo in a comment
- Use === rather than == for equality comparison
- Shorten function call code for click

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ module.exports = function() {
         {
           label: 'Quit',
           accelerator: 'Command+Q',
-          click: function() { app.quit(); }
```
Comment (carter-thaxton) [inline index.js:153]: Have you verified that this works?  It will change the binding of `this` on the call to `quit`.

Comment (paulbjensen) [inline index.js:153]: Hi, I tried it and it works, but you're probably right that it's safer to keep it in a function scope.

**Comment by paulbjensen** [comment]
I've changed the click function back to a scoped function, so the only changes now are the === for equality comparison and the change for it referencing the app via Electron's remote API.

**Comment by carter-thaxton** [comment]
Excellent.  Thanks!

**Comment by carter-thaxton** [comment]
Published to npm as 0.2.0

**Pull request merged**